src/lib/features/Header.svelte

```diff
@@ -5,32 +5,31 @@
   import { MenuItem } from '$lib/utils/types.js';
 
   export let menuItems: MenuItem[] = [];
-  export let logoType: 'primary' = 'primary';
-  // logoHex = teal
-  export let logoHex = '#55B1AB';
-
-  $: isNavOpen = false;
-
-  $: innerWidth = window.innerWidth;
-
-  $: hasMobileNav = innerWidth <= 744;
-
-  $: {
-    window.addEventListener('resize', () => {
-      innerWidth = window.innerWidth;
-      hasMobileNav = innerWidth <= 744;
-      if (!hasMobileNav) isNavOpen = false;
-    });
-  }
 </script>
 
-<div class={`flex justify-between bg-navy p-f16 md:bg-transparent md:p-0 ${$$restProps.class}`}>
+<div class={`flex h-[80px] justify-between p-f16 md:p-0 ${$$restProps.class}`}>
```

Comment:
Not a fan of the fixed header size, but it is what it is for now. Hoping to be able to remove it and let it stretch to fill